### DISCUSSION TITLE
Kolom kadastraalnummer bij BRK

### DISCUSSION THREAD

**Discussion by jhpoosthoek** [nlextract/NLExtract#391] [Ideas]
Nog een suggestie: de overal gebruikte kadastrale nummers zijn een combinatie van:
`concat("akrgemeente", '-', "sectie", '-', "perceelnummer")`
Is het misschien een idee om dit al als kadastraalnummer kolom in te bouwen?

**Comment by justb4** [comment]
Goede suggestie, dit heb ik al in verschillende projecten gedaan. Vooral om een 'fixed-length' key te genereren, met padding etc. Heb nog wel ergens SQL, maar misschien leuk voor jou om als PR toe te voegen. Dan promoveren we deze discussie naar Issue. 

Iets als:
```
coalesce(akrgemeente,'00000') || coalesce(sectie,'0') || lpad(coalesce(perceelnummer,'00000'),5,'0')
```

**Reply by jhpoosthoek** [reply]
Klinkt goed! Wat bedoel je met de afkorting PR?

**Reply by justb4** [reply]
Pull Request, de standaard manier om via GitHub bij te dragen.

**Reply by jhpoosthoek** [reply]
Ik gebruik git niet zo vaak (bijna nooit in een team situatie) dus denk dat het slimmer is als jij de PR doet ;-)

**Reply by justb4** [reply]
Hmm, denk mooie gelegenheid om je te verdiepen in GitHub. De software (NLExtract), is al gratis. Ik heb gratis ondersteuning gegeven. Ik heb hierboven een voorzet voor implementatie gegeven. Ik ga niet ook nog gratis ontwikkeling doen. Eventueel wel in opdracht. Dank voor je begrip!

**Reply by jhpoosthoek** [reply]
Snap ik, geen probleem, zo belangrijk is deze functionaliteit nou ook weer niet!

**Comment by pierregis** [comment]
Hi, begrijp ik goed dat "kadastraalnummer" geen gestandaardiseerd object is?
Soms zie ik een combinatie van "akrgemeente" ,"sectie", en "perceelnummer" met streepjes, of met spaties, en niet altijd met hetzelfde (perceelnummer) lengte...

**Reply by justb4** [reply]
Dat klopt, zie boven. Ieder project doet dat weer zelf. Wel met altijd zelfde lengte, perceelnr wordt aangevuld. Maar ook weer project afhankelijk...Na al die jaren zou dat best handig zijn als Kadaster dit zou standaardiseren, maar heeft met lange historie te maken. Zoals gezegd: PRs welkom!
